docker/ort-wrapper.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright (C) 2021 BMW CarIT GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+set -e
+
+#shellcheck disable=SC1091
```

Comment:
Change`#shellcheck disable=SC1091`to `# shellcheck disable=SC1091` aligned with usage in other files.